diff --git a/device/rockchip/common/scripts/mk-buildroot.sh b/device/rockchip/common/scripts/mk-buildroot.sh
--- device/rockchip/common/scripts/mk-buildroot.sh
+++ device/rockchip/common/scripts/mk-buildroot.sh
@@ -96,7 +96,7 @@ unset LD_LIBRARY_PATH
 
 touch "$BUILDROOT_OUTPUT_DIR/.stamp_build_start"
-if ! "$BUILDROOT_DIR"/utils/brmake -C "$BUILDROOT_DIR" O="$BUILDROOT_OUTPUT_DIR"; then
+if ! "$BUILDROOT_DIR"/utils/brmake -j$(( $(nproc) + 1 )) -C "$BUILDROOT_DIR" O="$BUILDROOT_OUTPUT_DIR"; then
 	error "Failed to build $BUILDROOT_BOARD:"
 	tail -n 100 "$LOG_FILE"
 	error "Please check details in $LOG_FILE"
