--- buildroot/package/rockchip/rockit/rockit.mk
+++ buildroot/package/rockchip/rockit/rockit.mk
@@ -12,13 +12,13 @@ ROCKIT_INSTALL_STAGING = YES
 ifneq ($(BR2_PACKAGE_RK3308),)
-ROCKIT_CONF_OPTS += -DRK3308=TRUE
-ROCKIT_DEPENDENCIES = alsa-lib
+ROCKIT_CONF_OPTS += -DRK3308=TRUE
+ROCKIT_DEPENDENCIES =
 
 else ifneq ($(BR2_PACKAGE_RK3506),)
 ROCKIT_CONF_OPTS += -DRK3506=TRUE
-ROCKIT_DEPENDENCIES = rockchip-rga alsa-lib
+ROCKIT_DEPENDENCIES = rockchip-rga
 
 else
-ROCKIT_DEPENDENCIES = rockchip-mpp rockchip-rga alsa-lib
+ROCKIT_DEPENDENCIES = rockchip-mpp rockchip-rga
 endif
