--- device/rockchip/common/overlays/rootfs/luckfox-lyra/S60netdevice
+++ device/rockchip/common/overlays/rootfs/luckfox-lyra/S60netdevice
@@ -28,7 +28,6 @@ start() {
 			echo "[S60netdevice] Saved current MAC address ($ETH0_CURRENT_MAC) to $ETH0_MAC_FILE"
 		fi
 
-		udhcpc -i eth0 > /tmp/eth0_udhcpc.log 2>&1 &
 	else
 		echo "[S60netdevice] eth0 not found"
 	fi
@@ -56,7 +56,6 @@ start() {
 			echo "[S60netdevice] Saved current MAC address ($ETH1_CURRENT_MAC) to $ETH1_MAC_FILE"
 		fi
 
-		udhcpc -i eth1 > /tmp/eth1_udhcpc.log 2>&1 &
 	else
 		echo "[S60netdevice] eth1 not found"
 	fi
